starter_code/index.js

```diff
@@ -1 +1,17 @@
 const Elevator = require('./elevator.js');
+const Person = require('./person.js');
+
+let elevator = new Elevator();
+let person1 = new Person("Lam", 0, 4);
+let person2 = new Person("Ngoc", 0, 4);
+
+elevator.call(person1);
+elevator.call(person2);
+elevator.floorUp();
+elevator.update();
+elevator.floorUp();
+elevator.update();
+elevator.floorDown();
+elevator.update();
+elevator._passengersLeave();
```

Comment:
Same for `_passengersLeave();` and `_passengersEnter();`
They should be called for you in your logic, by the `update` method